packages/govern-console/src/containers/DAO/DaoFinancePage.tsx

```diff
@@ -64,17 +88,26 @@ const ListTitle = styled.p`
 `;
 
 const DaoFinancePage: React.FC<Props> = ({ executorId, daoName, token: mainToken }) => {
-  const { provider } = useWallet();
+  const toast = useToast();
+  const { layoutName } = useLayout();
+  const { provider, isConnected } = useWallet();
+  const { data: finances, loading: isLoading } = useFinanceQuery(executorId);
+
   const [tokens, setTokens] = useState<FinanceToken>({});
-  const [isTransferModalOpen, setIsTransferModalOpen] = useState<boolean>(false);
   const [transactions, setTransactions] = useState<Transaction[]>([]);
+  const [isMediumPortrait, setMediumPortrait] = useState<boolean>(
+    layoutName === 'medium' && /portrait/.test(window.screen.orientation.type),
+  );
+  const [isTransferModalOpen, setIsTransferModalOpen] = useState<boolean>(false);
 
-  const { data: finances, loading: isLoading } = useFinanceQuery(executorId);
-  const context: any = useWallet();
-  const { isConnected } = context;
-  const { layoutName } = useLayout();
   const layoutIsSmall = useMemo(() => layoutName === 'small', [layoutName]);
-  const toast = useToast();
+
+  useEffect(() => {
+    window.addEventListener('resize', () =>
+      setMediumPortrait(layoutName === 'medium' && /portrait/.test(window.screen.orientation.type)),
+    );
+    return () => window.removeEventListener('resize', () => null);
```

Comment:
This does not work. You have to add the exact same function (as in define it outside and pass the same reference) to the remover as you passed to the adder.